src/form/Calendar/Calendar.tsx

```diff
@@ -292,8 +337,8 @@ export const Calendar: FC<CalendarProps> = ({
 };
 
 Calendar.defaultProps = {
-  previousArrow: '←',
-  nextArrow: '→',
+  previousArrow: '<',
```

Comment:
Why change this? It can be customized already.

Reply:
According to the new Figma design, I have made this change.

Reply:
this can be passed as a prop from your component. we shouldn't change it here as it's the default for others already using it.